templates/visuals/table/tsconfig.json

```diff
@@ -8,7 +8,7 @@
         "out": "./.tmp/build/visual.js"
     },
     "files": [
-        ".api/v1.1.0/PowerBI-visuals.d.ts",
+        ".api/v1.2.0/PowerBI-visuals.d.ts",
```

Comment:
again this shouldn't be the default version. 1.2 is in flux new visuals should still use 1.1 until 1.2 is stabilized.